manual/src/ornate/cookbook.md

```diff
@@ -325,3 +325,32 @@ You can use it e.g. with `[sbt-native-packager](https://github.com/sbt/sbt-nativ
 ~~~ scala
 mappings in (Compile, packageBin) ++= (webpack in (Compile, fullOptJS)).value.map { f => f.data -> f.data.getName() },
 ~~~
+
+## How to package for [Electron](https://electronjs.org)
+
+Packaging for Electron is actually not that hard either (but some more manual steps are needed to fully
+get this to work - e.g., writing Facades for Electron's built-in types.)
```

Comment:
I think we should focus on *how* to do things rather than discussing whether it’s hard or not. The part that you wrote between parens is the useful part, imho. Also, in the remaining subsections, you don’t give any clue for these facades. What about the following?

> Packaging for Electron requires to set the Webpack `target` configuration option to `electron`.